### ISSUE TITLE
Redundant sudo in Ubuntu setup script

### ISSUE THREAD

**Issue by tnlin** [getredash/redash#1581]
[Ubuntu setup script](https://github.com/getredash/redash/blob/master/setup/ubuntu/bootstrap.sh)
in Line 17, we verify user should run the script as root
I think Line 116/117's sudo is redundant ?
I would love to contribute from this small part (maybe virtualenv support next?)

**Comment by zodman** [comment]
related #1608

**Comment by r0fls** [comment]
I don't think this is redundant. This examples fails for me:

```bash
if [ "$(id -u)" != "0" ]; then
sudo /bin/bash $0
fi
touch /etc/tmp
```

However this works:

```bash
if [ "$(id -u)" != "0" ]; then
sudo /bin/bash $0
fi
sudo touch /etc/tmp
```

**Comment by arikfr** [comment]
Closed by #1704.
